<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="loginFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="login_frame_login">
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>0</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>登陆</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_frame_reset">
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>30</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_frame_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>0000000</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_frame_word">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="userFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="user_frame_welcomeMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>欢迎登陆！</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_frame_auth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_frame_logout">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭程序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>联系开发者</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>571</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>查询火车</string>
     </attribute>
     <widget class="QFrame" name="tab1_nologin">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;对不起，请先登录！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="tab1_logined">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>订票</string>
     </attribute>
     <widget class="QFrame" name="tab2_nologin">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;对不起，请先登录！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="tab2_logined">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>查询订单</string>
     </attribute>
     <widget class="QFrame" name="tab3_nologin">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;对不起，请先登录！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="tab3_logined">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>管理用户信息</string>
     </attribute>
     <widget class="QFrame" name="tab4_logined">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请输入用户ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tab4_logined_frame_userid">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>0000000</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tab4_logined_frame_usercheckid">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>查询ID信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>确认你的密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>电话</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tab4_logined_frame_username">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tab4_logined_frame_userword">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tab4_logined_frame_userword_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tab4_logined_frame_userphone">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tab4_logined_frame_usermail">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="tab4_logined_frame_mpdifyinfo">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>170</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>修改信息</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="tab4_nologin">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;对不起，请先登录！&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab5_aboutUs">
     <attribute name="title">
      <string>关于开发者</string>
     </attribute>
     <widget class="QFrame" name="about">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>561</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{border: 0px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>481</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;开发者：Peter Zheng&lt;/p&gt;&lt;p&gt;简单版火车票订票系统 0.1&lt;/p&gt;&lt;p&gt;由Qt引擎开发&lt;/p&gt;&lt;p&gt;联系方式：peterzheng98@sjtu.edu.cn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
